models/health.go

```diff
@@ -157,3 +163,10 @@ func (ps ProxyStatus) IsSynced() bool {
 func isComponentStatusSynced(componentStatus string) bool {
 	return componentStatus == "Synced"
 }
+
+func GetHealthConfiguration(annotations map[string]string) (string, error) {
+	if kialiConf, ok := annotations["health.kiali.io/rate"]; ok {
+		return kialiConf, nil
+	}
+	return "", fmt.Errorf("No Kiali Health configuration in Annotations")
```

Comment:
I don't think you should return an error here.
Not every Service would expect annotations.
So, I would send just empty string, then returning error may make no sense on this method (and all propagations).